composer.json

```diff
@@ -21,6 +19,7 @@
         "symfony/validator": "^4.3",
         "symfony/cache": "^4.3",
         "twig/twig": "^2.11",
+        "ezsystems/doctrine-dbal-schema": "^1.0@dev",
```

Comment:
We risk here that we'll forget about when tagging. Maybe we should just keep `"minimum-stability": "dev"` without `prefer-stable`?

Reply:
This is Composer ouput with that setting:
```
  - Installing ocramius/package-versions (dev-master 1d32342)
  - Installing symfony/polyfill-ctype (dev-master 550ebaa)
  - Installing symfony/yaml (4.4.x-dev 5c42497)
  - Installing psr/log (dev-master c4421fc)
  - Installing symfony/polyfill-php73 (dev-master 2ceb49e)
  - Installing symfony/polyfill-mbstring (dev-master b42a2f6)
  - Installing symfony/polyfill-php72 (dev-master 04ce333)
  - Installing symfony/polyfill-intl-idn (dev-master 6af626a)
  - Installing symfony/mime (dev-master 5fffd87)
  - Installing symfony/http-foundation (4.4.x-dev 8378bef)
  - Installing symfony/event-dispatcher-contracts (dev-master 6dd7d74)
  - Installing psr/container (dev-master 014d250)
  - Installing psr/cache (dev-master 78c5a01)
  - Installing symfony/event-dispatcher (4.4.x-dev c3fb43d)
  - Installing symfony/error-handler (dev-master c52d0b3)
  - Installing symfony/error-renderer (dev-master 9df8b91)
  - Installing symfony/http-kernel (4.4.x-dev bda78e4)
  - Installing symfony/service-contracts (dev-master 3cd78bb)
  - Installing symfony/dependency-injection (4.4.x-dev f684cb2)
  - Installing symfony/filesystem (4.4.x-dev 38b92b5)
  - Installing symfony/config (4.4.x-dev a1f6870)
  - Installing doctrine/event-manager (dev-master e05ae70)
  - Installing doctrine/cache (dev-master 17ec59c)
  - Installing doctrine/dbal (dev-master 29486f0)
  - Installing ezsystems/doctrine-dbal-schema (dev-master 6de4b25)
  - Installing twig/twig (2.x-dev c7b0a45)
  - Installing symfony/translation-contracts (dev-master 325b17c)
  - Installing symfony/validator (4.4.x-dev 6f98a2c)
  - Installing symfony/serializer (4.4.x-dev 281f577)
  - Installing psr/event-dispatcher (dev-master cfea31e)
  - Installing willdurand/jsonp-callback-validator (v1.1.0)
  - Installing symfony/console (4.4.x-dev 702aca4)
  - Installing symfony/routing (4.4.x-dev dcb7211)
  - Installing symfony/finder (4.4.x-dev 24c7afa)
  - Installing symfony/cache-contracts (dev-master 8ce952b)
  - Installing symfony/var-exporter (dev-master 12778a8)
  - Installing symfony/cache (4.4.x-dev 3a7a34f)
  - Installing paragonie/random_compat (v9.99.99.x-dev 0947f25)
  - Installing symfony/polyfill-php70 (dev-master 54b4c42)
  - Installing symfony/inflector (dev-master c3a425d)
  - Installing symfony/property-access (dev-master ca2eddb)
  - Installing symfony/security (4.4.x-dev bad9e46)
  - Installing doctrine/lexer (dev-master 6c4b4c3)
  - Installing doctrine/annotations (1.7.x-dev 3f35255)
  - Installing symfony/framework-bundle (4.4.x-dev 02916fa)
  - Installing friendsofsymfony/jsrouting-bundle (2.3.1)
  - Installing twig/extensions (1.x-dev 52fbe09)
  - Installing doctrine/reflection (dev-master 4a97a22)
  - Installing doctrine/collections (1.6.x-dev c5e0bc1)
  - Installing doctrine/persistence (1.2.x-dev 43526ae)
  - Installing doctrine/inflector (1.3.x-dev 45d9b13)
  - Installing symfony/doctrine-bridge (4.4.x-dev cfd57c3)
  - Installing doctrine/doctrine-cache-bundle (dev-master c46725e)
  - Installing jdorn/sql-formatter (dev-master 7ef9b85)
  - Installing doctrine/doctrine-bundle (1.12.x-dev 88e5ac9)
  - Installing psr/http-message (dev-master f6561bf)
  - Installing psr/http-factory (dev-master 12ac7fc)
  - Installing symfony/options-resolver (4.4.x-dev 35f472b)
  - Installing php-http/promise (dev-master 1cc44dc)
  - Installing psr/http-client (dev-master 496a823)
  - Installing php-http/httplug (dev-master d7f1b18)
  - Installing kriswallsmith/buzz (1.0.1)
  - Installing symfony/expression-language (4.4.x-dev ecc32c7)
  - Installing symfony/translation (4.4.x-dev 9d3130c)
  - Installing symfony/process (4.4.x-dev 1f31933)
  - Installing nikic/php-parser (dev-master a1f7269)
  - Installing jms/translation-bundle (dev-master 7bdd5e2)
  - Installing sensio/framework-extra-bundle (dev-master 197eb89)
  - Installing php-http/discovery (dev-master e822f86)
  - Installing clue/stream-filter (v1.4.1)
  - Installing php-http/message-factory (dev-master 597f30e)
  - Installing php-http/message (dev-master ce8f43a)
  - Installing php-http/client-common (dev-master 9a2a4c1)
  - Installing friendsofsymfony/http-cache (2.7.0)
  - Installing friendsofsymfony/http-cache-bundle (2.7.0)
  - Installing league/flysystem (dev-master 763cc64)
  - Installing oneup/flysystem-bundle (3.1.0)
  - Installing symfony/templating (4.4.x-dev 63f55de)
  - Installing symfony/asset (4.4.x-dev 3000a46)
  - Installing imagine/imagine (dev-develop dc16267)
  - Installing liip/imagine-bundle (2.1.0)
  - Installing zendframework/zend-code (dev-develop 8003f74)
  - Installing ocramius/proxy-manager (2.2.x-dev 41cac6e)
  - Installing pagerfanta/pagerfanta (v2.1.3)
  - Installing hautelook/templated-uri-router (2.0.x-dev 31ef14a)
  - Installing hautelook/templated-uri-bundle (2.1.0)
  - Installing nelmio/cors-bundle (dev-master 10a24c1)
  - Installing symfony-cmf/routing (dev-master c37797d)
  - Installing ezsystems/ezpublish-kernel (dev-master 3ba77f5)
  - Installing symfony/intl (dev-master cb7b25a)
  - Installing symfony/polyfill-intl-icu (dev-master 66810b9)
  - Installing symfony/form (4.4.x-dev 93e3941)
  - Installing ezsystems/repository-forms (dev-master 381443c)
  - Installing symfony/security-bundle (4.4.x-dev a1b7cbf)
  - Installing ezsystems/ezplatform-rest (dev-master e767226)
  - Installing phpunit/php-timer (dev-master 37d2894)
  - Installing sebastian/environment (dev-master 1c91ab3)
  - Installing sebastian/version (2.0.1)
  - Installing sebastian/resource-operations (dev-master d67fc89)
  - Installing sebastian/object-reflector (dev-master 3053ae3)
  - Installing sebastian/recursion-context (dev-master a58220a)
  - Installing sebastian/object-enumerator (dev-master 63e5a3e)
  - Installing sebastian/global-state (2.0.0)
  - Installing sebastian/exporter (dev-master 97cc7ae)
  - Installing sebastian/diff (dev-master d7e7810)
  - Installing sebastian/comparator (dev-master 9a1267a)
  - Installing phpunit/php-text-template (1.2.1)
  - Installing phpunit/php-file-iterator (dev-master 7f0f297)
  - Installing theseer/tokenizer (1.1.3)
  - Installing sebastian/code-unit-reverse-lookup (dev-master 5e86080)
  - Installing phpunit/php-token-stream (dev-master e899757)
  - Installing phpunit/php-code-coverage (6.1.4)
  - Installing doctrine/instantiator (dev-master 7c71fc2)
  - Installing webmozart/assert (1.4.0)
  - Installing phpdocumentor/reflection-common (1.0.1)
  - Installing phpdocumentor/type-resolver (0.4.0)
  - Installing phpdocumentor/reflection-docblock (4.3.1)
  - Installing phpspec/prophecy (dev-master 1927e75)
  - Installing phar-io/version (2.0.1)
  - Installing phar-io/manifest (dev-master 8c8bd1b)
  - Installing myclabs/deep-copy (1.x-dev 6c3e9f1)
  - Installing phpunit/phpunit (7.5.x-dev 062dfa4)
  - Installing matthiasnoback/symfony-config-test (dev-master 3087216)
  - Installing matthiasnoback/symfony-dependency-injection-test (v3.1.0)
  - Installing symfony/stopwatch (4.4.x-dev 3dfa8b3)
  - Installing php-cs-fixer/diff (v1.3.0)
  - Installing composer/xdebug-handler (1.3.3)
  - Installing composer/semver (dev-master 37e4db2)
  - Installing friendsofphp/php-cs-fixer (2.15.x-dev bc7f896)
```

I'd say that's too many packages, the risk of a breaking change in one of them seems high - we might have really unstable builds.

Also: I want to use solution we agree on in this repo in other repositories, also affected by this issue (AdminUI etc.), where the list of unstable packages used could be even higher.

Reply:
Ok, but then maybe it's enough to have it as dev-dependency, so it's kinda separated?

Reply:
I'm not really convinced - it is kinda separated, but:
- we still will have to revert that change before tagging
- `composer install --no-dev` won't work in this repository
- anybody trying to use this package in his own project will have to add his own dependency on `ezsystems/doctrine-dbal-schema` (this might be a bit far-fetched, but in theory it could happen)

But if you and @adamwojs think this is the best way I can change it

Reply:
> * `composer install --no-dev` won't work in this repository

Ok, if you need this, then fine by me :+1: